.jenkins/Jenkinsfile-s390x

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env groovy
+
+def lib
+// Currently the loadbalancer service is not working on minikube, same for network policies
+def DEFAULT_EXCLUDED_GROUPS = "flaky,loadbalancer,networkpolicies"
+
+pipeline {
+    //agent any
+    agent{
+        node {
+            label "strimzi-s390x"
+        }
+    }
+    tools {
+        maven 'maven-3.8.4'
+    }
+    parameters {
+        string(name: 'TEST_CASE', defaultValue: "", description: 'maven parameter for executing specific tests')
+        string(name: 'TEST_PROFILE', defaultValue: 'acceptance', description: 'maven parameter for executing specific test profile')
+        string(name: 'TEST_GROUPS', defaultValue: "", description: 'maven parameter for executing specific test groups')
+        string(name: 'EXCLUDE', defaultValue: "${DEFAULT_EXCLUDED_GROUPS}", description: 'maven parameter for exclude specific test tag')
+        string(name: 'ENV_VARS', defaultValue: "", description: 'maven parameter for setting values of environment variables used in STs')
+    }
+    options {
+        timeout(time: 24, unit: 'HOURS')
+        ansiColor('xterm')
+    }
+    environment {
+        ARTIFACTS_DIR = 'systemtest/target/logs'
+        JOB_NAME_SUB = "${String.format("%.15s", JOB_NAME).toLowerCase()}"
+        ORIGIN_BASE_DIR = "${env.WORKSPACE}/origin/"
+        KUBE_CONFIG_PATH = "${env.WORKSPACE}/origin/kube-apiserver/admin.kubeconfig"
+        OPERATOR_IMAGE_PULL_POLICY = "IfNotPresent"
+        COMPONENTS_IMAGE_PULL_POLICY = "IfNotPresent"
+        JAVA_VERSION = "11"
+        TEST_HELM2_VERSION = "v2.17.0"
```

Comment:
Is this still needed? I don't think we use Helm 2 for anything anymore.

Reply:
No, it is not needed. I've removed it.